src/app/featureFlags/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { FeatureFlagVariantValue } from './types';
+
+export const experiments: { [key: string]: FeatureFlagVariantValue[] } = {
+  searchButton: [
+    'originalButton',
+    'bannerColorButton',
+    'grayButton',
+  ],
+};
+
+export const experimentIds: { [key: string]: string } = {
```

Comment:
you could make the values here `keyof experiments` and then typescript will complain if there are any mismatches